src/Components/CreateImageWizard/formComponents/TargetEnvironment.js

```diff
@@ -67,6 +75,7 @@ const TargetEnvironment = ({ label, isRequired, ...props }) => {
                                 className='provider-icon'
                                 src={ '/apps/frontend-assets/partners-icons/microsoft-azure-short.svg' } /> }
                             onClick={ () => handleSetEnvironment('azure') }
+                            onKeyDown = { (e) => handleKeyDown(e, 'azure') }
```

Comment:
Can you also add the same the same on key handling for the checkbox. The tiles select with Space or Enter but the checkbox only selects with Space.

Reply:
It might actually be better to remove the ability to select the tiles with Enter and use Space for everything. Thoughts?

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/checkbox_role

It actually seems it is atypical for a checkbox to allow Enter, so I think going Space only might be best.

Reply:
Hahah was just reading the same. Yes lets go Space for everything. The tiles aren't visibly checkboxes but they function as checkbox.

Reply:
Those mozilla docs on accessibility are great :)

Reply:
https://webaim.org/techniques/keyboard/ is also helpful. University led accessibility project.